lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -176,6 +178,19 @@ def test_courseware_access(self):
             if result['error_code'] is not None:
                 self.assertFalse(result['has_access'])
 
+    def test_ignore_mobile_available_flag_access(self):
```

Comment:
The test here isn't needed since `TestUserEnrollmentApi` inherits from `MobileCourseAccessTestMixin`, which you already updated with this test.